### PR TITLE
Update VehicleMoveEvent.java

### PR DESCRIPTION
**Pull request by EvilPeanut**
Allow the vehicle move event to be cancelled

### CODE REVIEW

**Comment by EdGruberman** [comment]
Please follow the [PR formatting requirements](https://github.com/Bukkit/CraftBukkit/blob/master/CONTRIBUTING.md#pull-request-formatting-expectations).

You also have additional space characters that should not be present.

**Comment by turt2live** [comment]
@EvilPeanut if you are still interested in this please update it to the latest code base, resolve the mentioned errors in the code, and resolve the issues in the PR description and title as per the CONTRIBUTING guidelines.

A proper justification will need to be provided as well.

File: src/main/java/org/bukkit/event/vehicle/VehicleMoveEvent.java [hunk 1]
```diff
@@ -29,6 +31,19 @@ public Location getFrom() {
...
     /**
+     * Get if event is cancelled
+     *
+     * @return Event is cancelled
+     */
+   public boolean isCancelled() {
+        return cancelled;
+    }
+
+    public void setCancelled(boolean cancel) {
```
Comment (turt2live) [inline src/main/java/org/bukkit/event/vehicle/VehicleMoveEvent.java:42]: This needs JavaDocs

File: src/main/java/org/bukkit/event/vehicle/VehicleMoveEvent.java [hunk 2]
```diff
@@ -29,6 +31,19 @@ public Location getFrom() {
...
+        this.cancelled = cancel;
+    }
+
```
Comment (turt2live) [inline src/main/java/org/bukkit/event/vehicle/VehicleMoveEvent.java:45]: Extra spaces

**Comment by Wolvereness** [comment]
https://github.com/Bukkit/CraftBukkit/pull/1122#issuecomment-24468925

**Pull request closed without merging**